CONTRIBUTING.md

```diff
@@ -1,6 +1,27 @@
-# Thanks for Contributing!
+# Keen IO Javascript Client
 
-Run the following commands to get this project set up locally.
+First off, thank you for considering contributing to this official Keen IO client. It's people like you that make Keen IO such a great tool.
+
+We put these guidelines together to try and make working with our SDK as straight forward as possible, and hopefully help you understand how we communicate about potential changes and improvements.
+
+Improving documentation, bug triaging, building modules for various frameworks or writing tutorials are all examples of helpful contributions we really appreciate.
+
+Please, don't use the issue tracker for support questions. If you have a support question please come hang out in http://keen.chat or send an email to team@keen.io
+
+## Guidelines
+
+* Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
+* Be welcoming to newcomers and encourage diverse new contributors from all backgrounds. See the [Python Community Code of Conduct](https://www.python.org/psf/codeofconduct/).
+
+## Your First Contribution
```

Comment:
These resources are rad. It would be awesome to support folks' first contributions with some `first-timers-only` issues in whatever capacity we're able to.